controller-server/src/test/java/com/yahoo/vespa/hosted/controller/ControllerTest.java

```diff
@@ -696,7 +696,7 @@ void testDnsUpdatesForApplicationEndpoint() {
 
     @Test
     void testDevDeployment() {
-        ApplicationPackage applicationPackage = new ApplicationPackageBuilder().build();
+        ApplicationPackage applicationPackage = ApplicationPackageBuilder.fromDeploymentXml(DeploymentSpec.empty.xmlForm());
```

Comment:
How about no `deployment.xml` at all?

Reply:
It's fine, but I updated the test to use a completely empty package.